nmpolicy/internal/ast/types.go

```diff
@@ -43,6 +44,9 @@ func (n Node) String() string {
 	if n.EqFilter != nil {
 		return fmt.Sprintf("EqFilter(%s)", *n.EqFilter)
 	}
+	if n.NeFilter != nil {
+		return fmt.Sprintf("NeFilter(%s)", *n.EqFilter)
```

Comment:
Shouldn't it be `fmt.Sprintf("NeFilter(%s)", *n.NeFilter)`?

Reply:
Fixed